.github/workflows/main.yml

```diff
@@ -12,41 +12,40 @@ on:
 
 jobs:
   build-and-test:
-    name: Test PY=${{ matrix.python-version }}, DEV=${{ matrix.developer }}, EXP=${{ matrix.experimental }}, DEP=${{ matrix.deprecated }}
-    runs-on: ubuntu-latest
+    name: Test PY=${{ matrix.python-version }}, BCD=${{ matrix.bitcoind-version }}, EXP=${{ matrix.experimental }}, DEP=${{ matrix.deprecated }}
+    runs-on: ubuntu-22.04
```

Comment:
Why?

Reply:
I thought it would make sense to pin the image to a certain version to prevent surprises when `ubuntu-latest` gets updated to a newer version. But I changed my mind. It actually _does_ make sense if the tests stop passing for some reason with the latest Ubuntu :+1:
Fixed